### ISSUE TITLE
Current build fails...

### ISSUE THREAD

**Issue by adamretter** [javapathfinder/jpf-core#8]
So I have tried to follow the instructions for installing and building from source on the Wiki, but it doesn't work for me.

I am on Mac OS X and Oracle JDK 8.

My steps:
```bash
mkdir ~/code/jpf
cd ~/code/jpf
git clone https://github.com/javapathfinder/jpf-core.git
mkdir ~/.jpf

cat <<EOF >> ~/.jpf/site.properties
# JPF site configuration
jpf-core = ${user.home}/code/jpf/jpf-core
# numeric extension
jpf-numeric = ${user.home}/code/jpf/jpf-numeric
# annotation-based program properties extension
jpf-aprop = ${user.home}/code/jpf/jpf-aprop
extensions=${jpf-core},${jpf-aprop}
#... and all your other installed projects
EOF

cd ~/code/jpf/jpf-core
ant test
```

I then get these errors from Ant:
```
-compile-classes:
    [mkdir] Created dir: /Users/aretter/code/jpf/jpf-core/build/classes
    [javac] Compiling 84 source files to /Users/aretter/code/jpf/jpf-core/build/classes
    [javac] /Users/aretter/code/jpf/jpf-core/src/classes/sun/misc/SharedSecrets.java:61: error: cannot find symbol
    [javac]   private static JavaObjectInputStreamAccess javaObjectInputStreamAccess;
    [javac]                  ^
    [javac]   symbol:   class JavaObjectInputStreamAccess
    [javac]   location: class SharedSecrets
    [javac] /Users/aretter/code/jpf/jpf-core/src/classes/sun/misc/SharedSecrets.java:142: error: cannot find symbol
    [javac]   public static JavaObjectInputStreamAccess getJavaObjectInputStreamAccess() {
    [javac]                 ^
    [javac]   symbol:   class JavaObjectInputStreamAccess
    [javac]   location: class SharedSecrets
    [javac] /Users/aretter/code/jpf/jpf-core/src/classes/sun/misc/SharedSecrets.java:151: error: cannot find symbol
    [javac]   public static void setJavaObjectInputStreamAccess(JavaObjectInputStreamAccess access) {
    [javac]                                                     ^
    [javac]   symbol:   class JavaObjectInputStreamAccess
    [javac]   location: class SharedSecrets
    [javac] Note: Some input files use internal proprietary API that may be removed in a future release.
    [javac] Note: Recompile with -Xlint:sunapi for details.
    [javac] 3 errors
```

Seems like there are some source files missing?

**Comment by wvisser** [comment]
For what it is worth cloning, setting JAVA_HOME, and typing "ant" built fine for me. It simply gave warnings for the errors you found. I used the following JAVA_HOME setting on my Mac:
export JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_144.jdk/Contents/Home/

**Comment by jeandersonbc** [comment]
I'm also on Mac and using Oracle JDK 8 (1.8.0_151). I didn't have problems either.
@adamretter, have you checked your JAVA_PATH variable? Perhaps it is not properly set.

**Comment by wvisser** [comment]
Okay so here is what I found. On my Mac desktop, with jdk 1.8.0_144 it works. Then tried it on my laptop, running some other version, and it didn't work. I updated the jdk to the current version 161 and it still didn't work. then went down to 144 and it worked.

So this is not good at all, but it seems that 144 works for me, and from the other comments it seems 151 also works, but not the latest version 161.
